subsys/bluetooth/audio/Kconfig.has

```
@@ -0,0 +1,46 @@
+# Bluetooth Audio - Hearing Access Service options
+#
+# Copyright (c) 2022 Codecoup
+#
+# SPDX-License-Identifier: Apache-2.0
+
+config BT_HAS
+	bool "Hearing Access Service support [EXPERIMENTAL]"
+	select EXPERIMENTAL
+	help
+	  This option enables support for Hearing Access Service.
+
+if BT_HAS
+choice BT_HAS_HEARING_AID_TYPE_CHOICE
+	prompt "Hearing Aid Type selection"
+	help
+	  Select the Hearing Aid Type to compile.
+
+config BT_HAS_HEARING_AID_MONAURAL
+	bool "Monaural Hearing Aid"
+
+config BT_HAS_HEARING_AID_BINAURAL_LEFT
+	depends on BT_CSIS
+	bool "Left Binaural Hearing Aid"
+
+config BT_HAS_HEARING_AID_BINAURAL_RIGHT
+	depends on BT_CSIS
+	bool "Right Binaural Hearing Aid"
+
+config BT_HAS_HEARING_AID_BANDED
+	bool "Banded Hearing Aid"
+
+endchoice # BT_HAS_HEARING_AID_TYPE_CHOICE
+
+config BT_HAS_HEARING_AID_TYPE
+	int # hidden; ref HAS_d1.0r00; Table 3.2
+	range 0 2
+	default 0 if BT_HAS_HEARING_AID_BINAURAL_LEFT || BT_HAS_HEARING_AID_BINAURAL_RIGHT
+	default 1 if BT_HAS_HEARING_AID_MONAURAL
+	default 2 if BT_HAS_HEARING_AID_BANDED
+
```

Suggest to use the `help` field here. 

```suggestion
	int # hidden
	range 0 2
	default 0 if BT_HAS_HEARING_AID_BINAURAL_LEFT || BT_HAS_HEARING_AID_BINAURAL_RIGHT
	default 1 if BT_HAS_HEARING_AID_MONAURAL
	default 2 if BT_HAS_HEARING_AID_BANDED
	help
	  The value shall be one of 3 defined by the HAS 1.0 specification table 3.2

```